game-app/game-headed/src/main/java/org/triplea/game/client/HeadedGameRunner.java

```diff
@@ -187,10 +187,9 @@ private static void start() {
               () -> {
                 showMainFrame();
                 gameSelectorModel.setReadyForSaveLoad();
+                UpdateChecks.launch(headedServerSetupModel.getUi());
```

Comment:
`SwingUtilities.invokeLater` will run code on the "UI" thread.

From that thread, when we are done running code that needs to run on the "UI" thread, we run: `ThreadRunner.runInNewThread(`  will run more code in an otherwise ordinary thread.

So, we're "hopping" to the UI thread and back out. 

OTOH, what we had, would attempt to concurrently run Lines 181 & Lines 193 at the same time, ie:

> SwingUtilities.invokeLater(

and:

> UpdateChecks.launch(headedServerSetupModel.getUi());

Which is to say, there is no guarantee when, or even if the code from `SwingUtilities.invokeLater(` will be executed at all.